entities/player/Player.gd

```diff
@@ -38,17 +43,21 @@ func _physics_process(_delta):
 	_process_collisions()
 	move_and_slide()
 
+func damage_player(amount):
+	if is_invincible:
+		return
```

Comment:
I think godot typically uses pass but I could be wrong

Reply:
nah, pass is just a no op, return ends the function early